test.bash

```diff
@@ -0,0 +1,2 @@
+#/bin/bash
+yesno "Coucou"
```

Comment:
Don't commit that please

Reply:
yesno is defined here https://git.zx2c4.com/password-store/tree/src/password-store.sh

Reply:
I am an idiot

Reply:
Don't say that, I just did not make explicit enough the fact that this extension is sourced form password-store.sh. I may improve (and write) developer documentation.